test/widgets/inbox_test.dart

```diff
@@ -385,6 +450,50 @@ void main() {
           checkAppearsUncollapsed(tester, 1, findSectionContent);
         });
 
+        testWidgets('collapse stream section after scroll', (tester) async {
```

Comment:
Similarly to the other test description, let's mention briefly the expected outcome:
```suggestion
        testWidgets('collapse stream section when partially offscreen: header remains sticky at top', (tester) async {
```

That version also gets a bit more precise than "after scroll", which could mean a lot of different things — for example if we had some bug that was triggered by scrolling down, then scrolling back up to the same spot, then doing something, then a test for that bug's fix could equally well be described with "after scroll".